src/jit-format/jit-format.cs

```diff
@@ -164,7 +164,7 @@ private void validate()
                     {
                         Console.WriteLine("Formatting jit directory.");
                     }
-                    _srcDirectory = "jit";
+                    _srcDirectory = "src/jit";
```

Comment:
`Path.Combine("src", "jit");`

Reply:
Good point, thanks, fixed in 2nd commit.